source/documentation/administration_guide/topics/Cluster_Migration_Policy_Settings_Explained.adoc

```diff
@@ -50,3 +50,21 @@ The resilience policy defines how the virtual machines are prioritized in the mi
 |*Migrate only Highly Available Virtual Machines* |Migrates only highly available virtual machines to prevent overloading other hosts.
 |*Do Not Migrate Virtual Machines* |Prevents virtual machines from being migrated.
 |===
+
+.Additional Properties Settings
+[options="header"]
+|===
+|Field |Description/Action
+|*Enable Migration Encryption* a|Allows the virtual machine to be encrypted during migration.
+
+* Cluster default (Don’t encrypt)
```

Comment:
I see that it's also written that way elsewhere - but I think the `Don't encrypt` part should be dropped since it depends on the cluster's setting

Reply:
Removed (Don't Encrypt) from all locations.